packages/client/components/EstimatePhaseDiscussionDrawer.tsx

```diff
@@ -29,7 +30,12 @@ const Drawer = styled('div')<{isDesktop: boolean; isOpen: boolean}>(({isDesktop,
   transition: `all 200ms ${BezierCurve.DECELERATE}`,
   userSelect: 'none',
   width: isOpen || !isDesktop ? DiscussionThreadEnum.WIDTH : 0,
-  zIndex: ZIndex.SIDEBAR
+  zIndex: ZIndex.SIDEBAR,
+  [mediaQueryforPoker(Breakpoint.POKER_DISCUSSION_DRAWER)]: {
+    width: '100vw',
+    position: 'fixed',
+    right: isOpen ? '-360px' : '-100vw'
```

Comment:
-1 I know it works, but having -360px here (which should have been `DiscussionThreadEnum.WIDTH`) as right offset seems wrong. I think the logic should move here:
https://github.com/ParabolInc/parabol/blob/ab9ee86dcc4abea3815ff71f975ea1f1388de4ca/packages/client/components/PokerEstimatePhase.tsx#L25
instead of counteracting the width set there here.

Reply:
@Dschoordsch thanks for guidance but I am not able to figure out to do that.
Can you help me with this?

Reply:
I checked again in detail and it seems it's not that straight forward as I thought since the `SwipeableDasSidebar` defines the size and is used in other places as well.